apps/remote_control/lib/lexical/remote_control/search/store.ex

```diff
@@ -71,6 +75,10 @@ defmodule Lexical.RemoteControl.Search.Store do
     GenServer.start_link(__MODULE__, [project, create_index, refresh_index], name: __MODULE__)
   end
 
+  def start_link([create_index, refresh_index]) do
```

Comment:
This is a small thing, but something I noticed when I saw this being started in `RemoteControl.Application`: can we make this a kw list or a tuple?

```suggestion
  def start_link(opts) do
    create_index = Keyword.fetch!(opts, :create)
    refresh_index = Keyword.fetch!(opts, :update)
```

Reply:
i've always seen init_args as a list to mimic positional args. 🤔 

This seemingly small change has a fairly large change in tests and IMO, makes the ergonomics of `start_supervised!` a bit worse, now we'll have to type keywords into each call to it, where we can do something like, misspell keys (I did this very thing). 

If this was something that was a top-level API, i'd probably make that change, but I think that this only occurs in a couple places and it's not worth it.